### PR TITLE
add cmpopts dependency

### PR DESCRIPTION
**Pull request by rakelkar**
add dependency for #207 
```
dep ensure -add github.com/google/go-cmp/cmp/cmpopts
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/241)

### CODE REVIEW

**Comment by rakelkar** [comment]
@hougangliu there is a race causing tests to fail that i suspect is related to #229 ?
```
http://testing-argo.kubeflow.org/workflows/kubeflow-test-infra/kubeflow-kfserving-presubmit-e2e-241-388d12c-7841-5ec0?tab=workflow
```

File: Gopkg.toml [hunk 1]
```diff
@@ -81,3 +81,6 @@ version="v1.4.7"
 [[override]]
   name = "github.com/tensorflow/tensorflow"
   version = "=v1.13.1"
+
+[[constraint]]
+  name = "github.com/google/go-cmp"
```
Comment (yuzisun) [inline Gopkg.toml]: Can we specify the version ? Also seems it does not prune non-go and unused packages.

Comment (rakelkar) [reply Gopkg.toml]: Specified version

**Pull request merged**